src/templates/Header/mobile.tsx

```diff
@@ -0,0 +1,71 @@
+import { Container, createStyles, Hidden, IconButton, makeStyles, Theme } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import AddCircleOutlineIcon from '@material-ui/icons/AddCircleOutline';
+import PersonIcon from '@material-ui/icons/Person';
+import EventNoteIcon from '@material-ui/icons/EventNote';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    link: {
+      margin: `0 ${theme.spacing(1)}px`,
+    },
+    iconButton: {
+      width: '100%',
+      color: theme.palette.grey[50],
+    },
+    iconButtonLabel: {
+      display: 'flex',
+      flexDirection: 'column',
+    },
+    buttonGroup: {
+      width: 'fit-content',
+      display: 'grid',
+      gridTemplateColumns: 'repeat(3, 33.3%)',
+    },
+  })
+);
+
+const buttonsData = [
+  {
+    to: '/my-advertisements',
+    label: 'Dodane',
```

Comment:
In our project, this button is for rental items, I suggest moving that section to profile or modify the third menu item on something more general like 'menu' with the hamburger icon.